changelog_entry.yaml

```
@@ -1,4 +1,20 @@
 - bump: patch
   changes:
     added:
-    - Legal reference for NE CDCC and EITC.
\ No newline at end of file
+    - Legal reference for NE CDCC and EITC.
```

Remove everything below this and fix what looks like a merge conflict via `git pull upstream master`
```suggestion
    - Legal reference for Nebraska CDCC, EITC, AGI reduction by SSB, and extra standard deduction.
```

---

Hi Max, what does make format do? I typed the command on github and all it says is "black . -l 79"

Also, I am assuming git pull upstream master updates my current codespace from the original source. I am not 100% sure though. Do you have any recommendations when it comes to figuring out how exactly codespace works?

---

`make format` formats the code via `black` and `linecheck` - you can see what it does in `Makefile`

codespaces is an IDE, but yes `git pull upstream master` syncs your branch with the policyengine/policyengine-us master branch. looks like you still have a merge conflict in changelog_entry.yaml, which you should be able to resolve by clicking "accept current" when opening that file